### ISSUE TITLE
Json support

### ISSUE THREAD

**Issue by benvenutti** [pirovc/ganon#23]
Question: would it be interesting for the user to be able to use a json file for the setup (as an alternative to the command line arguments)?

**Comment by pirovc** [comment]
Interesting idea, would be nice to have such feature but it is not a priority neither a standard for tools like ganon. I would do it only after a first release, since the arguments may still change until then.
